src/engine/math_util.c

```diff
@@ -27,6 +27,11 @@ Vec3s gVec3sOne  = {     1,     1,     1 };
 
 static u16 gRandomSeed16;
 
+// Generate a pseudorandom integer from 0 to 255.
+u8 random_u8(void) {
```

Comment:
this seems pretty useless imo, if it is kept i think it should be set to always inline at the very least

Reply:
fair actually, especially considering additional bias. This would only be useful in cases of needing exactly a power of 2, which still offers no benefit whatsoever to runtime _and_ usually still means using a modulo. I vote remove here as well.